code/modules/jobs/access.dm

```diff
@@ -31,6 +36,9 @@
 			return TRUE
 	return FALSE
 
+/obj/proc/allow_buckled_access(mob/accessor)
```

Comment:
This proc should handle the logic for checking access of buckled mobs, so that the behaviour can be overriden in a more fine way rather than just enabled/disabled.